pkg/cli/mode/navigation.go

```diff
@@ -1,6 +1,6 @@
-// Package navigation provides the functionality of navigating the filesystem.
 package mode
 
+// This module provides the navigation mode widget for navigating the filesystem.
```

Comment:
This is some leftover when the nav mode was in its own module. Just remove this.

Reply:
Done